simple-facebook-instant-articles.php

```diff
@@ -111,8 +112,21 @@ public function template_redirect() {
 	 * @return void
 	 */
 	public function render( $post_id ) {
+
 		do_action( 'simple_fb_pre_render', $post_id );
-		include( apply_filters( 'simple_fb_article_template_file', $this->template_path . '/article.php' ) );
+
+		if ( have_posts() ) {
+
+			the_post();
+
+			$template = apply_filters( 'simple_fb_article_template_file', $this->template_path . '/article.php' );
```

Comment:
@dashaluna I've added this here - VIP requested it elsewhere, but I think they must have missed it here. Since we're touching this bit of code, lets fix.